src/controllers/dlgprefcontroller.cpp

```diff
@@ -21,6 +21,17 @@
 #include "preferences/usersettings.h"
 #include "util/version.h"
 
+QString nameForPreset(const PresetInfo& preset);
```

Comment:
Oh, sorry, I had to put the whole body of the nameForPreset function in the namespace, I only put the prototype in the namespace, I correct it, sorry